docs/en/query_language/agg_functions/reference.md

```diff
@@ -514,6 +514,55 @@ Calculates the average.
 Only works for numbers.
 The result is always Float64.
 
+## avgWeighted {#avgweighted}
+
+Calculates the weighted arithmetic mean.
+
+**Syntax** 
+
+```sql
+avgWeighted(x, weight)
+```
+
+**Parameters** 
+
+- `x` — Values to be calculated. 
+- `weight` — Weights of the values.
+
+Type of all the arguments must be the same.
+
+**Returned value**
+
+- If not all the weights are equal to 0, function returns [Integer](../../data_types/int_uint.md) or [Float*](../../data_types/float.md) value.
+- If all the weights are equal to 0, function returns `NaN`.
+
```

Comment:
Type: [integer](../../data_types/int_uint.md) or [floating-point](../../data_types/float.md).

Is it true? check by `toType(avgWeighted(x, weight))` function.